### PR TITLE
chore: upload all the logs from /tmp

### PR DESCRIPTION
**Pull request by romange**

### CODE REVIEW

File: .github/workflows/regression-tests.yml [hunk 1]
```diff
@@ -53,7 +53,7 @@ jobs:
         uses: actions/upload-artifact@v4
         with:
           name: logs
-          path: /tmp/failed/*
+          path: /tmp/*
```
Comment (chakaz) [inline .github/workflows/regression-tests.yml:56]: maybe limit to `/tmp/dragonfly.*`? there could be other, unrelated files under `/tmp`, no?

Comment (romange) [reply .github/workflows/regression-tests.yml:56]: so we will see, I am running regression tests to test that

**Review by chakaz — APPROVED** [review]

**Review by chakaz — DISMISSED** [review]

**Review by chakaz — APPROVED** [review]
